libs/test-setup/src/connectors.rs

```diff
@@ -15,6 +15,7 @@ fn connector_names() -> Vec<(&'static str, Tags)> {
         ("postgres", Tags::POSTGRES),
         ("postgres11", Tags::POSTGRES),
         ("postgres12", Tags::POSTGRES),
+        ("postgres13", Tags::POSTGRES),
```

Comment:
I'm kind of surprised it's enough to define a new connector for the whole test macros infrastructure, I didn't remember I made it that easy :p

Reply:
There are a few test api functions also that were a bit of a drag to add...

Reply:
Yeah, but I think these can't be avoided if we want control.

Reply:
Maybe we could reduce the boilerplate for these with some `macro_rules!` magic, or even a full-on proc macro.

Reply:
If we start testing more versions of databases, for example.